client/src/locales/en.json

```diff
@@ -1,5 +1,234 @@
+  "common.member-details.sack.user-roles-member": "Can view, add and edit sack's items.",
+  "common.member-details.cohort.user-roles-member": "Can view sacks, add and edit sack's items.",
+  "common.member-details.cohort.user-roles-owner": "Can edit, archive and delete this cohort. Can add, edit sack's items and mark them as done. Can add, edit, archive and delete sacks. Can add, remove members, and change their roles.",
+  "common.member-details.remove": "Remove user",
+  "common.member-details.role-member": "member",
+  "common.member-details.role-owner": "owner",
+  "common.member-details.role-viewer": "viewer",
+  "common.member-details.role-guest": "(Guest)",
+  "common.members-form.email": "email",
+  "common.members-form.add": "add",
+  "common.page-404.heading": "Oops!",
+  "common.page-404.error": "Error code: 404 (page not found)",
+  "common.page-404.message": " It looks that the page you are trying to reach might have been removed or it is temporarily unavailable.",
+  "common.page-404.dashboard": "Go to dashboard",
+  "common.sort-box.sort": "sort {sortOrder}",
+  "common.tile-view-item.remove-fav": "Remove from favourites",
+  "common.tile-view-item.add-fav": "Add to favourites",
+  "common.breadcrumbs.cohorts": "cohorts",
+  "common.breadcrumbs.dashboard": "dashboard",
+  "common.members.actions.invitation": "Invitation email to: {data} successfully sent.",
+  "common.members.actions.invitation-fail": "Something went wrong while sending invitation email. Please try again.",
+  "about.heading": "App history",
+  "about.content-1": "Inspiration to create the End of coffee application was born out of the need to manage our office equipment. In the initial phase, it was to be only an internal training project for new employees. By participating in the project, developers were free to choose technology and tools. The End Of Coffee application can also be an interesting point in the portfolio of every employee who participated in the process of creating this product.",
+  "about.content-2": " The main functionality of the application are the sacks on which we can place products that are currently missing in the company. Depending on the role of the user on the sack, he can add new products, vote for them, mark them as ordered and comment. As part of the application, you can also create cohorts that can help you organize and group your sacks. The visibility of sacks within a cohort depends on the sack of members of a given team. In the future, we plan to make the application an open-source project and anyone who wants to contribute to the project.",
+  "about.try-cta": "Try End of coffee app today!",
+  "about.cta-button": "Go to app",
+  "about.intro-heading": "End of coffee Office inventory tracking app",
+  "about.intro-subheading": "End of coffee helps you track all the office inventory you need!",
+  "authorization.auth-box.heading": "End of coffee",
```

Comment:
common.app-name: "End of Coffee" and use it as parameter in other strings

Reply:
how to manage this?
I can't pass two id's to 
<FormattedMessage /> component or to formatMessage() function.

https://stackoverflow.com/questions/34876767/how-to-deal-with-multiple-usage-of-a-react-intl-message

Reply:
You can do sth like this:
`"about.intro-heading": "{appName} Office inventory tracking app"`
`"app.name": "End of coffee"`
`<FormattedMessage id="about.intro-heading" values={{ appName: formatMessage({ id: "app.name" })}} />`